### PR TITLE
Add simplicity to the styles

### PR DESCRIPTION
**Pull request by platosha**
Connected to #6 

This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-split-layout/25)

### CODE REVIEW

File: vaadin-split-layout.html [hunk 1]
```diff
@@ -90,10 +79,9 @@
         @apply(--layout-vertical);
       }
 
-      :host > ::content > *:not(#splitter) {
+      :host ::content > * {
```
Comment (limonte) [inline vaadin-split-layout.html]: This is what you mentioned in https://github.com/vaadin/vaadin-split-layout/pull/21, right?

Comment (platosha) [inline vaadin-split-layout.html]: Yep.

**Comment by manolo** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-split-layout/25)*

**Comment by manolo** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-split-layout/25)*

**Pull request merged**